apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/content/staticelement/PageStaticController.java

```diff
@@ -94,4 +100,35 @@ public DeleteStaticResponse deleteStaticElement(
                 : new DeleteStaticResponse("delete fail");
     }
 
+    @PutMapping("/{id}/update/hyperlink")
```

Comment:
question:  Is `update` in the path redundant as `PUT` already implies that it is an update?  This is a @mpeels call